### ISSUE TITLE
Use a noop reducer

### ISSUE THREAD

**Issue by zalmoxisus** [romseguy/redux-devtools-chart-monitor#15]
I see the reducer is handling `TOGGLE_VISIBILITY`, but it is not dispatched from anywhere. Is it an orphaned code that should be removed?

**Comment by romseguy** [comment]
Legacy code yes, redux-devtools wasn't implementing visibility state as far as I remember. I don't think we care too much about bundle size since it's a dev tool, so maybe we should just use a `noop` [here](https://github.com/romseguy/redux-devtools-chart-monitor/blob/master/src/ChartMonitor.js#L37) instead of removing the reducer and actions files, could make potential future improvements more straightforward... We have git history of course but the convenience here looks free from drawbacks to me.

**Comment by zalmoxisus** [comment]
Of course, if we'll use a noop function and those files won't be imported, it wouldn't affect the bundle size as well. I am working on adding support for monitors reducers into Redux DevTools Extension, and was wondering about extra operations.
